packages/builder/src/components/backend/DataTable/modals/CreateEditColumn.svelte

```diff
@@ -143,6 +144,8 @@
       )?.[0]
       // Set the tableId based on the selected table
       editableColumn.tableId = relationshipTableIdSecondary
+    } else if (editableColumn.subtype === USER_TYPE) {
```

Comment:
Don't we need the `USERS` type as well?

Reply:
The picker does default to many, but it probably is worth being explicit.

Would multiple users be `many-to-one` or `many-to-many` in this case?

Reply:
users type added